src/components/HeroHeader/index.jsx

```diff
@@ -8,14 +8,23 @@ import { makeStyles } from 'cozy-ui/transpiled/react/styles'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 import { isTwakeTheme } from 'cozy-ui/transpiled/react/helpers/isTwakeTheme'
 
-const useStyles = makeStyles(theme => ({
-  nameInverted: {
-    textShadow: theme.textShadows[1]
-  },
-  hostInverted: {
-    textShadow: theme.textShadows[1]
+const useStyles = makeStyles(theme => {
+  return {
+    title: {
+      letterSpacing: 0.35,
```

Comment:
you maybe have this value in `theme`, inside `theme.typography` maybe? But it's suprising to override a Typography value 🤔

Reply:
After checking deeper, 0.35 is not already existing in `theme.typography`. And no `letterSpacing` is defined for h1, h2, h3, ... It seems like an edge case.